backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -4,118 +4,234 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 
-import { createTestClient } from 'apollo-server-testing'
-import createServer from '@/server/createServer'
-import CONFIG from '@/config'
+import { Community as DbCommunity } from '@entity/Community'
+import { testEnvironment } from '@test/helpers'
 
-jest.mock('@/config')
+// jest.mock('@/config')
 
 let query: any
 
 // to do: We need a setup for the tests that closes the connection
 let con: any
+let testEnv: any
 
 beforeAll(async () => {
-  const server = await createServer({})
-  con = server.con
-  query = createTestClient(server.apollo).query
+  testEnv = await testEnvironment()
+  query = testEnv.query
+  con = testEnv.con
+  await DbCommunity.clear()
 })
 
 afterAll(async () => {
   await con.close()
 })
 
 describe('CommunityResolver', () => {
-  const getCommunityInfoQuery = `
+  const getCommunities = `
```

Comment:
please outsource to `seeds/graphql/queries.ts`

Reply:
done